### ISSUE TITLE
Clicking NEUS fall/spring switches the fall/spring indicator, but doesn't switch the map (or reset the species selection). Same for West coast, Southeast.

### ISSUE THREAD

**Issue by mpinsky** [Ecotrust/oceanadapt#70]
In the Northeast US, West Coast, or Southeast regions, there are multiple options (e.g., fall/spring, annual/triennial, or spring/summer/fall). Choosing one, then choosing a species, shows the correct map and graphs.

However, subsequently clicking the fall/spring (or related) options switches the fall/spring visual indicator, but does not update the maps and graphs. 

As an example, this map is showing the spring survey, despite the indicator on the left now showing fall:

![image](https://user-images.githubusercontent.com/6370532/54249656-60140580-4517-11e9-8302-30f15e03dd15.png)

**Comment by pollardld** [comment]
### Reason for issue

Regional data for each species has a unique ID. Queries for each species regional data require knowing the unique ID

### Description  

queries Each species has a different id for each region.
Each row contained in the `species_table` of the database includes the following (*e.g.*, lumpfish):  

| species auto id | region name | species name | species common name |
| --------------- | ----------- | ------------ | ------------------- |  
| 694 | Northeast US Fall | Cyclopterus lumpus | lumpfish |
| 750 | Northeast US Spring | Cyclopterus lumpus | lumpfish |  

The species option list is generated using only one of the two ids, whichever the user selects. When a user has selected a species and then tries to toggle fall and spring the query is using the same species id as the original query. 

The user would need to select the species from the species list again in order to get an updated species id. Which is how the current site overcomes this issue.


### Options  

- [ ] Remove the option to toggle between fall and spring when viewing species data
- [ ] Take user back to one step and make them select species again from new list of species with updated species auto ids. Screen shot of one step back:  
  <img width="1582" alt="Screen Shot 2019-03-15 at 2 04 42 PM" src="https://user-images.githubusercontent.com/2632164/54462006-772b4100-472b-11e9-8e07-58d6d0fd128d.png">
- [ ] Major overhaul of PHP & MySQL

**Comment by mpinsky** [comment]
I think the most natural user interface (beside a major overhaul) would be to take the user back one step and make them select a species again. Otherwise, they need to go all the way back to the home page, right?

**Comment by pollardld** [comment]
Yes, a user would go back home. We should discuss more tomorrow.

**Comment by mpinsky** [comment]
This is better, but my thought was that fall/spring (or spring/summer/fall or annual/trienniel, depending on region) would stay available as options. Clicking the other option would deselect the species (as if you went back one step):

![image](https://user-images.githubusercontent.com/6370532/55344119-8f64c500-547a-11e9-8847-95b94f2ecfb9.png)
